cmd/notation/list.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
maybe rename the file to "signature-list" to be specific

Reply:
notation is about signature.